main/webapp/modules/core/scripts/project/history-panel.js

```diff
@@ -165,6 +170,22 @@ HistoryPanel.prototype._render = function() {
   this.resize();
 };
 
+HistoryPanel.prototype.showHideFacetSplash = function(clicked) {
+  var hideHistoryFacetSplash = JSON.parse(Refine.getPreference("ui.history.hideFacetSplash", false));
```

Comment:
```suggestion
  var hideHistorySplash = JSON.parse(Refine.getPreference("ui.history.hideFacetSplash", false));
```
- Facet and History at the same time? 
- And, also, same for JSON as before
- The name of the preference is wrong if you want 2 separate prefs.

These have implication at other places. I won't specify them, please look for them.